src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Descriptor/builder_priority_tag.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container>
+  <definition id="definition_3" class="Full\Qualified\Class3" public="true" synthetic="true" lazy="false" shared="true" abstract="false" autowired="false" autoconfigured="false" file="/path/to/file">
+    <tags>
+      <tag name="tag1">
+        <parameter name="attr1">val1</parameter>
+        <parameter name="attr2">val2</parameter>
+        <parameter name="priority">0</parameter>
+      </tag>
+      <tag name="tag1">
```

Comment:
Shouldn't this `tag1` with priority `40` be displayed before the other `tag1` with priority `0`?

Reply:
Thanks for your question. 
Indeed, this method sorts services by priority tags but not tag itself. If a service have the same tag twice but with different priority, sorting take in consideration uniquely the tag with the superior priority.

Should I make a change for fix that ?